.github/release/release_notes.md

```diff
@@ -0,0 +1,85 @@
+# v0.1.8
+
+## `--list-main-classes` option for the `run` command
+
+You can pass the option `--list-main-classes` to the `run` command to list all the available main classes, including
+scripts.
+
+```
+$ scala-cli . --list-main-classes
+Hello scripts.AnotherScript_sc scripts.Script_sc
+```
+
+Added in [#1095](https://github.com/VirtusLab/scala-cli/pull/1095) by @Gedochao
+
+## Add `config` command
+
+The `config` sub-command allows to get and set various configuration values, intended for use by
+other Scala CLI sub-commands.
+
+This feature has been added in preparation for the `publish` command, stay tuned for future announcements.
+
+Added in [#1056](https://github.com/VirtusLab/scala-cli/pull/1056) by @alexarchambault
+
+## Prioritise non-script main classes
+
+When trying to run a directory containing scripts and just a single non-script main class, the non-script main class
+will now be prioritised and run by default.
+
+```
+$ scala-cli .
+Running Hello. Also detected script main classes: scripts.AnotherScript_sc, scripts.Script_sc
+You can run any one of them by passing option --main-class, i.e. --main-class scripts.AnotherScript_sc
+All available main classes can always be listed by passing option --list-main-classes
+Hello world
+```
+
+Changed in [#1095](https://github.com/VirtusLab/scala-cli/pull/1095) by @Gedochao
+
+## Important bugfixes
+
+### Enable downloading Scala versions outside the Coursier cache
+
+Scala CLI is now able to download the newest Scala versions, even if they're not yet available in the Coursier cache. In
```

Comment:
"even if they're not yet available in the Coursier cache" doesn't sound right to me (we used to have a problem if they're not in the Scala version listing in the cache). I'd say it like this instead:

Scala CLI uses version listings from Maven Central to check if a Scala version is valid. When new Scala versions are released, users could sometimes have stale version listings in their Coursier cache for a short period of time (the Coursier cache TTL, which is 24 hours by default). This prevented these users to use new Scala versions.
To work around that, Scala CLI now retries to download version listings when they don't have the requested Scala version.